examples/lessons/12/app.js

```diff
@@ -0,0 +1,206 @@
+      .then(textures => {
+        let cube = new Cube(gl, {
+          vs: VERTEX_SHADER,
+          fs: FRAGMENT_SHADER,
+          uniforms: {
+            uSampler: textures[0]
+          }
+        });
+        return {moon, cube};
+      });
+    });
+  },
+  onRender: ({
+    gl, tick, aspect, moon, cube
+  }) => {
+    // Update Camera Position
+    const eyePos = [0, 0, 20];
+
+    gl.clear(GL.COLOR_BUFFER_BIT | GL.DEPTH_BUFFER_BIT);
+
+    let uMoonMMatrix = new Matrix4()
+      .multiplyRight(moonRotation.matrix);
+    let uCubeMMatrix = new Matrix4()
+      .multiplyRight(cubeRotation.matrix);
+    let uVMatrix = new Matrix4()
+      .lookAt({eye: eyePos, center: [0, 0, 0], up:[0, 1, 0]});
+
+    let element = null;
+    let lighting = (element = document.getElementById("lighting")) ? element.checked : true;
+
+    moon.setUniforms({uUseLighting: lighting});
+    cube.setUniforms({uUseLighting: lighting});
+
+    if (lighting) {
+      let ambientColor = new Vector3(
+        parseFloat((element = document.getElementById("ambientR")) ? element.value : "0.2"),
+        parseFloat((element = document.getElementById("ambientG")) ? element.value : "0.2"),
+        parseFloat((element = document.getElementById("ambientB")) ? element.value : "0.2")
+      );
+
+      let pointLightingLocation = new Vector3(
+        parseFloat((element = document.getElementById("lightPositionX")) ? element.value : "0"),
+        parseFloat((element = document.getElementById("lightPositionY")) ? element.value : "0"),
+        parseFloat((element = document.getElementById("lightPositionZ")) ? element.value : "0")
+      );
+
+      let pointLightColor = new Vector3(
+        parseFloat((element = document.getElementById("pointR")) ? element.value : "0.8"),
+        parseFloat((element = document.getElementById("pointG")) ? element.value : "0.8"),
+        parseFloat((element = document.getElementById("pointB")) ? element.value : "0.8")
+      );
+
+      moon.setUniforms({
+        uAmbientColor: ambientColor,
+        uPointLightingLocation: pointLightingLocation,
+        uPointLightingColor: pointLightColor
+      });
+
+      cube.setUniforms({
+        uAmbientColor: ambientColor,
+        uPointLightingLocation: pointLightingLocation,
+        uPointLightingColor: pointLightColor
+      });
+    }
+
+    moon.render({
+      uMMatrix: uMoonMMatrix,
+      uVMatrix,
+      uPMatrix: new Matrix4().perspective({fov: 45 * Math.PI / 180, aspect, near: 0.1, far: 100})
+    });
+
+    cube.render({
+      uMMatrix: uCubeMMatrix,
+      uVMatrix,
+      uPMatrix: new Matrix4().perspective({fov: 45 * Math.PI / 180, aspect, near: 0.1, far: 100})
+    });
+    animate(timeline, moonRotation, cubeRotation);
+  }
+});
+
+animationLoop.getInfo = () => {
+  return `
+  <p>
+    <a href="http://learningwebgl.com/blog/?p=1359" target="_blank">
+    Point lighting
+    </a>
+  <p>
+    The classic WebGL Lessons in luma.gl
+    `;
+};
+
+function degToRad(degree) {
```

Comment:
math.gl exports a function `radians` 

`const rad = radians(degrees)`

Reply:
Good to know. Done.